README.md

```diff
@@ -71,108 +73,147 @@ To compile and build the `broadinstitute/firecloud-ui` docker image, run
 ./script/build.sh compile -d build
 ```
 
+## IntelliJ
+
+There are effectively two projects in this repository: a ClojureScript project for the
+FireCloud UI and a Scala project for the Selenium-based UI tests. They are logically
+linked because the tests rely on the rendered DOM structure (primarily `data-test-id`
+attributes) but are otherwise largely separate. Therefore (also in part because of
+[https://youtrack.jetbrains.com/issue/SCL-12358]()) we recommend opening
+separate IntelliJ projects for each.
+
+For the ClojureScript project, choose `File` -> `Open` and select the base `firecloud-ui`
+directory of your git clone. If prompted for a Project JDK, select a recent Java SDK to
+use. Assuming you have the Cursive plugin installed, IntelliJ will automatically detect
+the `project.clj` file and configure the CLJS Leiningen project.
+
+For the Scala project, choose `File` -> `Open` and select the `automation` directory
+inside the git clone. Select a recent Java SDK (i.e. 1.8) and click `OK`. IntelliJ will
+automatically detect the `build.sbt` file and configure the Scala/SBT project. You will
+need to accept the suggestion to add the git root. In your firecloud-ui clojure project
+you may want to choose the automation directory and set it to ignore. If you add a new
```

Comment:
In your firecloud-ui clojure project you may want to choose the automation directory and set it to ignore. **Otherwise,** if you add a new file in the scala project and choose not to add it to the git repo, you may be asked again when you switch over to the clojure project.